core/src/main/java/io/seata/core/store/db/AbstractDataSourceProvider.java

```diff
@@ -195,6 +202,18 @@ protected String getUser() {
      */
     protected String getPassword() {
         String password = CONFIG.getConfig(ConfigurationKeys.STORE_DB_PASSWORD);
+        String publicKey = getPublicKey();
+        if (StringUtils.isNotBlank(publicKey)) {
+            try {
+                password = ConfigTools.decrypt(publicKey, password);
```

Comment:
ConfigTools is the class of druid. Is any problem when use other? And this is the abstract class, I think you should override the method of 
```
getPassword()
```

Reply:
ok